lib-release.js

```diff
@@ -22,6 +23,12 @@ const processArgs = process.argv.slice(2)
 
 process.stdin.setEncoding('utf8');
 
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  terminal: false
```

Comment:
May we don't need to set this to `false`. But let's see:
https://nodejs.org/api/readline.html#readline_readline_createinterface_options